deebot_client/hardware/deebot/snxbvc.py

```
@@ -0,0 +1 @@
+x5d34r.py
```

Ruff (0.8.0)

Found useless expression. Either assign it to a variable or remove it.

(B018)